doc/text/news.md

```diff
@@ -1,5 +1,19 @@
 # News
 
+## 1.1.1: 2015-02-13
+
+### Improvements
+
+  * {Groonga::Command::LogicalRangeFilter}: Added.
+    [GitHub#4] [Patch by Hiroshi Hatake]
+  * {Groonga::Command::LogicalCount}: Added.
+    [GitHub#5] [Patch by Hiroshi Hatake]
+
+### Fixes
+
+  * Fix return type in yard doc.
```

Comment:
Umm, I don't understand fully what you means 'Use `Fixed` like `Added`'.
You mean following two things?

* Uniform document style 
* Use YARD instead of "yard"

Thus, this document should be fix as follows:

```
{Return type in YARD doc}: Fixed.      
```